### ISSUE TITLE
Use custom font for language name on target card header.

### ISSUE THREAD

**Issue by cckozie** [unfoldingWord-dev/ts-android#2129]
Same issue as we had on the source language select and select for download screens.
![screenshot_2017-04-07-15-35-56](https://cloud.githubusercontent.com/assets/8171759/24819156/17da3684-1ba9-11e7-92aa-cf9e2ff10e63.png)

**Comment by PhotoNomad0** [comment]
Good catch, the title should be using the same font as the translation.

**Comment by PhotoNomad0** [comment]
Fixed in PR 2132
